core/src/main/java/com/aem/demo/core/servlets/ServletForRssFeed.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
This copyright info surely doesn't belong here. You're not Adobe, and it's not 2015 ;)))

Reply:
Seriously speaking, copyright notices are added when a project is licensed under some particular conditions. This happens quite often. But if they are added, they are accompanied by a license file in the repository root, and are then prepended to each and every file. 
You will see this when you join our open-source solutions. But for your sandbox project, copyright notices are a bit irrelevant